skeleton/Rakefile

```diff
@@ -1,5 +1,6 @@
 require 'puppetlabs_spec_helper/rake_tasks'
-require 'puppet/vendor/semantic/lib/semantic'
+require 'puppet/version'
+require 'puppet/vendor/semantic/lib/semantic' unless Puppet.version.to_f < 3.6
```

Comment:
I believe it's because symantic isn't included in Puppet code until version 3.6. https://github.com/puppetlabs/puppet/tree/master/lib/puppet/vendor/semantic
Which, I'm not sure, but I don't think has a standalone installation